<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Git</class>
 <widget class="QMainWindow" name="Git">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Git</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 73, 116)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32, 73, 116)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_inicio">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="btn_iniciarSesion">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>420</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 172, 56);
font-family: ebrima;
color:rgb(255, 255, 255);
font-size: 22px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Iniciar sesión</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: ebrima;
color:rgb(255, 255, 255);
font-size: 38px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Bienvenido a Git Papus</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>250</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: ebrima;
color:rgb(255, 255, 255);
font-size: 24px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Usuario:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>340</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: ebrima;
color:rgb(255, 255, 255);
font-size: 24px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_registrar">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>470</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 172, 56);
font-family: ebrima;
color:rgb(255, 255, 255);
font-size: 22px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Registrarse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_usuario">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font-family: ebrima;
color:rgb(0, 0, 0);
font-size: 16px;
font-weight: bold;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_password">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>340</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font-family: ebrima;
color:rgb(0, 0, 0);
font-size: 18px;
font-weight: bold;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_registro">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>350</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: ebrima;
color:rgb(255, 255, 255);
font-size: 24px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_usuarioRegistro">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font-family: ebrima;
color:rgb(0, 0, 0);
font-size: 16px;
font-weight: bold;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="text_passwordRegistro">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>350</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font-family: ebrima;
color:rgb(0, 0, 0);
font-size: 18px;
font-weight: bold;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>260</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: ebrima;
color:rgb(255, 255, 255);
font-size: 24px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Usuario:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>160</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: ebrima;
color:rgb(255, 255, 255);
font-size: 38px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Registro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_registrarse">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>440</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 172, 56);
font-family: ebrima;
color:rgb(255, 255, 255);
font-size: 22px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Registrarse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_atrasRegistro">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>490</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 172, 56);
font-family: ebrima;
color:rgb(255, 255, 255);
font-size: 22px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Atrás</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
